client/assets/sass/styles.sass

```diff
@@ -4,7 +4,6 @@
 @import '_bem-support/index'
 
 // Variables: colors, sizes, ...
-@import 'colors'
 @import 'app_colors'
 @import 'app_urls'
 
```

Comment:
So the issue here is that we are heavily reliant on requiring the sass files in a specific order when we do not need to anymore. With webpack we can now treat all of our assets as modules and explicitly require them where they are used. This makes it far easier to understand where things are coming from and also whether or not they are being used, thus leading to more maintainable code.

Also it is necessary to do this in order to import the sass directly to the components that use them such as in: 

https://github.com/ManageIQ/manageiq-ui-service/pull/519/files#diff-2a87b2c4b10a6970565a04a140ae0345R1